tests/client.py

```diff
@@ -655,6 +656,43 @@ def get_workflows(self):
                         if record[replication_key] >= self.start_date])
         return records
 
+    def _get_tickets_by_pk(self, ticket_id):
+        """
+        Get a specific ticket by pk value
+        HubSpot API https://developers.hubspot.com/docs/api/crm/tickets
+        """
+        url = f"{BASE_URL}/crm/v4/objects/tickets/{ticket_id}?associations=contact,company,deals"
+        response = self.get(url)
+        return response
+
+    def get_tickets(self):
+        """
+        Get all tickets.
+        HubSpot API https://developers.hubspot.com/docs/api/crm/tickets
+        """
+        url = f"{BASE_URL}/crm/v4/objects/tickets"
+        replication_key = list(self.replication_keys["tickets"])[0]
+        records = []
+
+        # response = self.get(url)
+
+        params = {"limit": 100, "associations": "contact,company,deals"}
+        while True:
+            response = self.get(url, params=params)
+
+            records.extend(
+                [
+                    record
+                    for record in response["results"]
+                    if record[replication_key] >= self.start_date_strf.replace('.Z', '.000Z')
+                ]
+            )
```

Comment:
```suggestion
            records.extend([record
                            for record in response["results"]
                            if record[replication_key] >= self.start_date_strf.replace('.Z', '.000Z')])
```

Reply:
This code has mix coding styles but for list comprehession lets use above style like used elsewhere in the same file.

Reply:
done